environments.json

```diff
@@ -3233,13 +3233,13 @@
     ]
   },
   "graalvm": {
-    "full": "GraalVM JavaScript 19.0.0",
+    "full": "GraalVM JavaScript 20.0.0",
```

Comment:
can we keep the results for v19?

Reply:
I've separated the GraalVM 19.3.1 and GraalVM 20.0.0 results. Not much difference at the moment, but we will make GraalVM 20.1.0 by default ES 2020, then the 19 and 20 results will differ more significantly.